hardware_interface/include/hardware_interface/loaned_command_interface.hpp

```diff
@@ -51,6 +60,27 @@ class LoanedCommandInterface
 
   virtual ~LoanedCommandInterface()
   {
+    auto logger = rclcpp::get_logger(command_interface_.get_name());
+    RCLCPP_WARN_EXPRESSION(
```

Comment:
Should we put this to "DEBUG" mode?

Or do we always want output when switching controllers?

Reply:
This is only when we deactivate the controller, i think it's good to have this statistical information.

Moreover, as it is conditioned it should only print for async cases and that too if you have some misses. This should give us some information of the functionality of the system